addons/test_event_full/tests/test_wevent_register.py

```diff
@@ -42,3 +42,27 @@ def test_register(self):
         self.assertEqual(visitor.partner_id, self.env['res.partner'])
         self.assertEqual(visitor.mobile, "0456112233")
         self.assertEqual(visitor.email, "raoulette@example.com")
+
+    def test_event_update_lead(self):
```

Comment:
Can't the test be added in website_event_crm ?

Reply:
Rather hectic, cause I don't have access to EventQuestion from there

Reply:
@nd-dew actually it seems we have a specific bridge module (`website_event_crm_question`) with a similar test ` test_event_registration_lead_description` in 16 so I agree it may be best to put it there instead